docs/security.md

```diff
@@ -47,6 +48,17 @@ follows:
 * Import all exported public keys into a single trust-store
 * Distribute the trust-store over the nodes
 
+### Configuring SASL Encryption
+
+To enable SASL encryption for an application, set `spark.authenticate.enableSaslEncryption` to
```

Comment:
Yeah. It would be redundant, slightly, but I was just thinking that it wouldn't hurt for people who skip around in this document or deep-link to it.